mammoth/opts.py

```diff
@@ -1013,14 +1014,15 @@ def _add_train_dynamic_data(parser):
         "--pool_size",
         type=int,
         default=2048,
-        help="Number of examples to dynamically pool before batching.",
+        help="(Maximum) number of examples to dynamically pool before batching.",
     )
     group.add(
         "-n_buckets",
         "--n_buckets",
         type=int,
-        default=1024,
-        help="Maximum number of bins for batching.",
+        default=4,
+        help="The number of minibatches that will be yielded once bucketing is complete. "
+        "Recommended value: same as accum_count, or at least a multiple of it."
```

Comment:
bucket is a terrible name for that?

Reply:
Bucket is indeed a terrible name. I reused the existing parameter without renaming it, while totally changing its meaning. Classic mammoth.

The plan was to either clean this up or throw this away based on the benchmark results, but unfortunately they were somewhat inconclusive.